### DISCUSSION TITLE
Relationship Field to a POD with Autocomplete, Single Select, Add New & Taggable options unable to accept Numeric Only Entries

### DISCUSSION THREAD

**Discussion by papa-pup** [pods-framework/pods#6790] [Ideas]
## Issue Overview
Relationship Field to a POD with Autocomplete, Add New & Taggable options unable to accept Numeric Only Entries

## Expected Behavior

 

1. The POD with the Relationship Field should have the data entry
2. The Related POD should have the data entry created

## Current Behavior

 

1. The POD with the Relationship Field have no data entry
2. The Related POD have no data entry

## Steps to Reproduce (for bugs)

1. Create a POD - i.e. PostalCode with NO extra fields. Changes to default PODs settings are Advanced Options -> Default Status -> Published.
2. Create another POD - i.e. Location with a Relationship field to PostalCode. Changes to Field Default settings are Single Select, AutoComplete, Add New & Taggable selected. Changes to PODs settings are Advanced Options -> Default Status -> Published. 
3. Create a page with shortcode [pods name="LOCATION" form="1" fields="POST_CODE"]
4. Testing Scenario
- Enter "12345" via form, yields no creation of entry on PostalCode POD
- Enter "12345 ABCDE" via form, yields creation of entry on PostalCode POD 

## WordPress Environment

<details>
WordPress Version: 5.3.2
PHP Version: 7.2.7
MySQL Version: 5.6.41
Server Software: LiteSpeed
Your User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36
Session Save Path: /opt/alt/php72/var/lib/php/session
Session Save Path Exists: No
Session Save Path Writeable: No
Session Max Lifetime: 1440
Opcode Cache:
- Apc: No
- Memcached: No
- OPcache: No
- Redis: No
Object Cache:
- APC: No
- APCu: No
- Memcache: No
- Memcached: No
- Redis: No
WPDB Prefix: wp_
WP Multisite Mode: No
WP Memory Limit: 40M
Current Memory Usage: 45.615M
Current Memory Usage (real): 46.250M
Pods Network-Wide Activated: No
Pods Install Location: /home/XXXX/YYYYYYY/wp-content/plugins/pods/
Pods Tableless Mode Activated: No
Pods Light Mode Activated: No
Currently Active Theme: Total
Currently Active Plugins:
- Address Autocomplete Using Google Place API: 1.0.0
- Code Snippets: 2.13.3
- Health Check & Troubleshooting: 1.4.2
- Pods - Custom Content Types and Fields: 2.7.16.2
- Shortcodes Ultimate: 5.6.2
- Wordfence Security: 7.4.2
- WP Reset: 1.77
- WP Rollback: 1.7.0
</details>

## Pods Package Export (helpful!)

Migrate: Packages NOT ENABLED

## Workaround or Alternate Solution Until Bug is Addressed

Yet Discovered.

## Related Issues and/or PRs
Not of any known so far.

[a comment by an automated account is not shown]

**Comment by JoryHogeveen** [comment]
@sc0ttkclark any ideas?

**Comment by papa-pup** [comment]
@JoryHogeveen , this is more of a **bug** than an **enhancement** type isn't it?
